resources/osl_gitlfs.rb

```diff
@@ -0,0 +1,21 @@
+resource_name :osl_gitlfs
+
+default_action :sync
+
+provides :osl_gitlfs
+property :directory, String,  required: true
+property :repository, String, required: true
+
+action :sync do
+  run_context.include_recipe 'base::gitlfs'
+  run_context.include_recipe 'git'
```

Comment:
Also, this should be run before ``base::gitlfs``
```suggestion
  run_context.include_recipe 'osl-git'
```

Reply:
You should go ahead and also remove including the ``git`` recipe since that is taken care of in ``osl-git``.